design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -57,3 +59,8 @@ type CheckoutPaymentMethod {
 type CartGiftCard {
     code: String!
 }
+
+type CartError {
+    identifier: String
```

Comment:
Please provide a use case when `identifier` is needed on storefront.

It seems like it should not be exposed to storefront. If so than this `CartError` type is excessive and its usages can be replaced with `String`

Reply:
The main reason to add the identifier is localizing part of process where errors were thrown. It should fetch profit for developers who extend base functionality and implements new features.